vnext/ReactUWPTestApp/Consts.ts

```diff
@@ -0,0 +1,21 @@
+export const APP_NAME = 'ReactUWPTestApp'
+
+// Framework
+export const SEARCH_BUTTON = 'SearchButton'
+export const HOME_BUTTON = '_HomeButton'
+export const REACT_CONTROL_ERROR_TEST_ID = 'ReactControlErrorMessage'
+
+// UnknownTestPage
+export const UNKNOWN_TESTPAGE = 'UnknownTestPage'
+
+// TextInputTestPage
+export const TEXTINPUT_TESTPAGE = 'TextInputTestPage'
+
+export const TEXTINPUT_ON_TEXTINPUT = 'TextInput'
+
+// LoginTestPage
+export const LOGIN_TESTPAGE = 'LoginTestPage'
+export const USERNAME_ON_LOGIN = 'UserName'
+export const PASSWORD_ON_LOGIN = 'Password'
+export const SUBMIT_ON_LOGIN = 'Submit'
+export const LOGINRESULT_ON_LOGIN = 'Result'
```

Comment:
Having one central file for string constants that span all our tests won't scale very well.  Can this be changed so that shared strings are defined in the test itself?